CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contribution to .NET Monitor
+
+You can contribute to .NET Monitor with issues and PRs. Simply filing issues for problems you encounter is a great way to contribute. Contributing implementations is greatly appreciated.
+
+## Reporting Issues
+
+We always welcome bug reports, proposals and overall feedback. Here are a few tips on how you can make reporting your issue as effective as possible.
+
+### Finding Existing Issues
+
+Before filing a new issue, please search our [open issues](https://github.com/dotnet/dotnet-monitor/issues) to check if it already exists.
+
+If you do find an existing issue, please include your own feedback in the discussion. Do consider upvoting (👍 reaction) the original post, as this helps us prioritize popular issues in our backlog.
+
+### Writing a Good Bug Report
+
+Good bug reports make it easier for maintainers to verify and root cause the underlying problem. The better a bug report, the faster the problem will be resolved. Ideally, a bug report should contain the following information:
+
+* A high-level description of the problem.
+* A _minimal reproduction_, i.e. the smallest size of code/configuration required to reproduce the wrong behavior.
+* A description of the _expected behavior_, contrasted with the _actual behavior_ observed.
+* Information on the environment: OS/distro, CPU arch, SDK version, etc.
+* Additional information, e.g. is it a regression from previous versions? are there any known workarounds?
+
+### DOs and DON'Ts
+
+Please do:
+
+* **DO** follow our coding style.
```

Comment:
The original doc linked to a style guide to follow - is that something we should add, or is that unnecessary for now?